ScrapyKeeper/config.py

```diff
@@ -1,14 +1,16 @@
 # Statement for enabling the development environment
 import os
+from ScrapyKeeper.ParameterStore import ParameterStore
 
 DEBUG = True
 
 # Define the application directory
 
 
 BASE_DIR = os.path.abspath(os.path.dirname(__file__))
+ssm_client = ParameterStore()
```

Comment:
Rename this variable to be more descriptive.
```suggestion
param_store = ParameterStore()
```